src/views/SingleEventView.vue

```diff
@@ -209,6 +222,13 @@ export default {
     closeEventModal() {
       this.openModal = false;
     },
+    openAnnouncementModal(announcement) {
+      this.announcementModalOpen = true;
+      this.modalAnnouncement = announcement;
+    },
+    closeAnnouncementModal() {
+      this.announcementModalOpen = false;
```

Comment:
I don't think it's super necessary since it'll just be reassigned when a new one is opened, but I changed it anyways since it can't hurt :).  Should I change it in the ProfileView page too, since I based this code off of that?

Reply:
If it's not necessary, I say we just leave it out. Because it's reassigned at every open we shouldn't need to do anything with it on close.